content/participants/ttjc@harshasalim/index.md

```diff
@@ -53,6 +53,24 @@ _stack_ Made using React Native and Firebase.
 
 _github link_ https://github.com/Hostel-Bucket
 
+#### Perfect Plan
+
+_description_ A social media web app, that allows users to formulate the perfect plan. This enables users to to post a plan, and have others suggest modifications to refine it.  
+
+_stack_ Made using ReactJs, Express and Firebase.
+
+_github link_ https://github.com/harshasalim/perfect-plan
+
+_hosted link_ https://perfect-plan-a0e83.web.app/
+
+#### Game of Life
```

Comment:
provide the hosted link to your project

Reply:
Sorry, I do not want to include that project as part of the competition.

Reply:
Few things to focus:
- Projects and blogs done after mid-March 2020 are accepted for the challenge.
- Each project mentioned should have `description` `hosted link` and `repository link` mentioned
- So please update accordingly and remove the past ones.
Regards,
Kanishk Gupta